R/helpers.R

```diff
@@ -117,8 +117,7 @@ use_rd_macros <- function(package) {
 
   if (!package %in% macros) {
     macros <- c(macros, package)
-    macros <- paste0("    ", macros, collapse = ",\n")
-    desc::desc_set(RdMacros = macros, file = proj, normalize = TRUE)
+    desc::desc_set(RdMacros = macros, file = proj)
```

Comment:
I see

```
   [ OK: 438 | SKIPPED: 21 | WARNINGS: 1 | FAILED: 1 ]

  1. Failure: use_lifecycle() respects existing RdMacros field 
```

Why this change?

Reply:
Ooops, I missed that test — but setting `normalize = TRUE` normalises the entire `DESCRIPTION` which feels a bit aggressive